recipes/soci/all/conanfile.py

```diff
@@ -115,14 +115,27 @@ def _configure_cmake(self):
         self._cmake.configure()
         return self._cmake
 
+    def _find_file_case_insensitive(self, libname, path='.'):
+        lower_filename = "find{}.cmake".format(libname.lower())
+        matched_files = [filename for filename in os.listdir(path) if filename.lower() == lower_filename]
+        if len(matched_files) == 0:
+            raise Exception("File {filename} not found in {directory} despite of case incensitive search".format(filename=lower_filename, directory=path))
+        if len(matched_files) > 1:
+            raise Exception("File {filename} matched multiple times in {directory} during case incensitive search".format(filename=lower_filename, directory=path))
+
+        m = re.search("find(.*)\\.cmake", matched_files[0], re.IGNORECASE)
+        return m.group(1), matched_files[0]
+
     def _patch_sources(self):
         for patch in self.conan_data.get("patches", {}).get(self.version, []):
             tools.patch(**patch)
 
         if self.options.with_backend_sqlite3:
-            tools.replace_in_file("FindSQLite3.cmake", "SQLite3_FOUND", "SQLITE3_FOUND")
-            tools.replace_in_file("FindSQLite3.cmake", "SQLite3_INCLUDE_DIRS", "SQLITE3_INCLUDE_DIRS")
-            tools.replace_in_file("FindSQLite3.cmake", "SQLite3_LIBRARIES", "SQLITE3_LIBRARIES")
+            (sqlite3, find_sqlite3_cmake) = self._find_file_case_insensitive("SQLite3")
```

Comment:
@madebr I have a strange behavior here, FindSQLite3.cmake on windows and Findsqlite3.cmake on linux. Any advice ?

Reply:
Use `FindSQLite3.cmake`.
Just 3 days ago, the name was fixed: https://github.com/conan-io/conan-center-index/commit/17fb12eebb3db591ddd696d5a73e7bd4145e6bff

You probably have an old package in your conan cache. Run `conan remove sqlite3` to force downloading an updated version on your next build.

Reply:
Works fine now, thank you !